benchmarks/query/bench_aiobotocore.py

```diff
@@ -0,0 +1,36 @@
+import asyncio
```

Comment:
Hmmm a module docstring would be nice :)
Should it return something?
The way the timing information is passed... somewhere... is just so odd 😅

Reply:
i've updated the readme quite a bit, hope that helps.